client/constants.py

```diff
@@ -1,3 +1,5 @@
+MAX_DESC_LENGTH = 200
```

Comment:
actually maybe this should be in config? cause they can change this value if they want